cli/src/commands/help.rs

```diff
@@ -47,3 +73,52 @@ pub(crate) fn cmd_help(
 
     Err(command_error::cli_error(help_err))
 }
+
+#[derive(Clone)]
+struct Keyword {
+    name: &'static str,
+    description: &'static str,
+    content: &'static str,
+}
+
+// TODO: Add all documentation to keywords
+//
+// Maybe adding some code to build.rs to find all the docs files and build the
+// `KEYWORDS` at compile time.
+//
+// It would be cool to follow the docs hierarchy somehow.
+//
+// One of the problems would be `config.md`, as it has the same name as a
+// subcommand.
+//
+// TODO: Find a way to render markdown using ANSI escape codes.
+//
+// Maybe we can steal some ideas from https://github.com/martinvonz/jj/pull/3130
+const KEYWORDS: &[Keyword] = &[
+    Keyword {
+        name: "revsets",
+        description: "A functional language for selecting a set of revision",
+        content: include_str!("../../../docs/revsets.md"),
```

Comment:
Does anyone know if this pointer outside of the crate will work when we publish to crates.io (as I'm supposed to do on wednesday)?

Reply:
Good catch. It could be worked around by a symlink (`ln -s ../docs cli`), but symlink has its own problem, so I have no idea.

Reply:
I somewhat doubt it will, I found a rather [confusing thread](https://users.rust-lang.org/t/include-str-does-not-work-when-releasing-because-of-changed-pathes/15551) about it.

We should probably instead use `build.rs` in order to get access to `OUT_DIR` and just copy the files during build time somewhere. A minor annoyance but nothing too horrid, I guess (though I'd need to figure out how to port the Buck changes.)

Reply:
Do you mean that we should make `build.rs` copy the files from `docs/` into e.g. `cli/srcs/docs/` so we have two copies of the files?

Reply:
Perhaps, what we can do with `build.rs` is to fall back to `../docs` if `./docs` doesn't exist? We'll need `docs` symlink or something in order to include `docs/*` in the release package.